src/editor_transport.rs

```diff
@@ -127,8 +125,6 @@ pub fn start_unix(path: &path::Path, sender: Sender<EditorRequest>) {
                             continue;
                         }
                         debug!("From editor: {}", request);
-                        let request: EditorRequest =
-                            toml::from_str(&request).expect("Failed to parse editor request");
                         if sender.send(request).is_err() {
                             return;
                         };
```

Comment:
> This generally indicates an internal error with kak-lsp with escaping in
> the toml-serializing shell code. A fatal error is not ideal here because
> it brings down all language servers and causes requests to fail during
> the kak-lsp and language server startup process.

Huh, I thought there's only one kak-lsp "controller" process for each language
server and that's it. Apparently there's also a central kak-lsp process that
listens on the session socket. Not sure. Anyway, this seems fine.